templates/launch.html

```diff
@@ -93,9 +94,10 @@
         <select id="language_code" name="language_code" class="qa-language-code">
             <option name="en" value="en">en</option>
             <option name="cy" value="cy">cy</option>
+            <option name="" value="">null</option>                            
```

Comment:
Are we happy to use `null` as the user string here?

Reply:
Not really, what about something like `<not set>`?